nevergrad/functions/games/game.py

```diff
@@ -195,116 +194,117 @@ def flip_value(self, visible1, visible2, l1, l2, stack, policy1):
-
-        p = Position(board=EMPTY_BOARD, ko=None)
-
-        mixing = 3  # We mix 3 policies. Please note that we also apply a 8-fold random rotation/symmetry.
-        if policy1 is not None:
-            init = np.random.randint(mixing)
-            policy1 = [policy1[i] for i in range(init, len(policy1), mixing)]
-        if policy2 is not None:
-            init = np.random.randint(mixing)
-            policy2 = [policy2[i] for i in range(init, len(policy2), mixing)]
-
-        history1 = 1  # Player 1 starts.
-        history2 = 2
-
-        random_rot = np.random.randint(8)
-
-        def transformation(x):
-            a = x // size
-            b = x % size
-            if random_rot == 0:
-                a = size - 1 - a
-                b = size - 1 - b
-            elif random_rot == 1:
-                b = size - 1 - b
-            elif random_rot == 2:
-                a = size - 1 - a
-                b = size - 1 - b
-            elif random_rot == 3:
-                b = size - 1 - b
-            elif random_rot == 4:
-                c = b
-                b = a
-                a = c
-                a = size - 1 - a
-            elif random_rot == 5:
-                c = b
-                b = a
-                a = c
-            elif random_rot == 6:
-                c = b
-                b = a
-                a = c
-                a = size - 1 - a
-            elif random_rot == 7:
-                c = b
-                b = a
-                a = c
-            return a * size + b
-
-        # pylint: disable=broad-except
-        for _ in range(2 * size * size - size - 1):
-            # print("move " + str(idx))
-            # print("=================")
-            # print(str(p))
-            # print("=================")
-            for _ in range((size * size - 9) // 2):
-                try:
-                    move1 = transformation(self.phantomgo_choose(policy1, history1))
-                    # print("player1 trying ", move1)
-                    p = p.play_move(move1, board.BLACK)
-                    history1 = 2 * history1 + 1  # legal move.
-                    break
-                except Exception:
-                    # print("failed!" + str(e))
-                    history1 = 2 * history1 + 2  # illegal move.
-            # print("=================")
-            # print(p)
-            # print("=================")
-            for _ in range((size * size - 9) // 2):
-                try:
-                    move2 = self.phantomgo_choose(policy2, history2)
-                    # print("player 2 trying ", move2)
-                    p = p.play_move(move2, board.WHITE)
-                    history2 = 2 * history2 + 1  # legal move.
-                    break
-                except Exception:
-                    # print("failed!" + str(e))
-                    history2 = 2 * history2 + 2  # illegal move.
-
-        return 1 if p.score() > 0 else 2
+    # TODO remove if not planned to be used
```

Comment:
Is the following planned to be used (phantomgo)? there are many undefined variables so I commented it, but if possible we should remove it